client/login/index.js

```diff
@@ -17,7 +17,7 @@ export default router => {
 
 	if ( config.isEnabled( 'login/wp-login' ) ) {
 		router(
-			'/log-in/:twoFactorAuthType(authenticator|backup|sms|push)?/:lang?',
+			'/log-in/:twoFactorAuthType(authenticator|backup|sms|push)?/:socialConnect(social-connect)?/:lang?',
```

Comment:
It looks like a boolean for now (value is there or absent).
I agree we could find a more general term for it such as `:flow`.